### PR TITLE
Cleanup feature flag "friends by online status"

### PR DESCRIPTION
**Pull request by Antivortex**
## What does this PR change?

This PR removes the usage of remote feature flag friends_by_online_status and modifies friends list UI to only keep implementation of this feature flag ON option (friends divided in two lists by online / offline status)

## How to test the changes?

Check if friends lists work well

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by sandrade-dcl — CHANGES_REQUESTED** [review]
As this PR changes a lot of things on files (including prefabs!) that we are currently also modifying in the Channels branch (https://github.com/decentraland/unity-renderer/pull/2553), it is going to be  a pain to resolve the conflicts from this branch into the Channels one.

I suggest to create a sub-branch from `feat/channels` and apply this changes there, be based on the current channels changes. I know it will force @Antivortex to re-apply the work and this would also postpone the merging of this change until we release Channels, but I think it is worth it in order to avoid conflicts.

What do you think @aixaCode?

Apart from that, for future tasks assignations, if it is something related to Chat & Friends, lets analyze first if it is needed to take into account the current Lazy Loading/Channels implementation or if we can apply directly the fix on `dev`.

**Comment by aixaCode** [comment]
@sandrade-dcl Thanks for raising this up. Good point, we should be more careful when doing changes to Chat and Friends section for future.
However, with this case, it looks like all prefabs changes are serialization changes from the TextMeshPro update. We could probably discard them, and not include them in this PR. That way we could just merge those changes to dev, and there will not be any conflict with channels.
@Antivortex Are the changes to prefab really needed? Could we revert those to avoid conflict with `feat/channels`.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDComponentView.cs
Comment (Antivortex) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDComponentView.cs:96]: I agree

**Comment by Antivortex** [comment]
> @sandrade-dcl Thanks for raising this up. Good point, we should be more careful when doing changes to Chat and Friends section for future. However, with this case, it looks like all prefabs changes are serialization changes from the TextMeshPro update. We could probably discard them, and not include them in this PR. That way we could just merge those changes to dev, and there will not be any conflict with channels. @Antivortex Are the changes to prefab really needed? Could we revert those to avoid conflict with `feat/channels`.

Yes, those changes in the prefab only contain removal of redundant game objects, I think those can wait for later after chat changes will be merged too and for now we can just disable those game objects in the code.

**Review by sandrade-dcl — APPROVED** [review]

**Pull request merged**
